machine_management/capi-machine-management.adoc

```diff
@@ -0,0 +1,109 @@
+:_content-type: ASSEMBLY
+[id="capi-machine-management"]
+= Managing machines with the Cluster API
+include::_attributes/common-attributes.adoc[]
+:context: capi-machine-management
+
+toc::[]
+
+:FeatureName: Managing machines with the Cluster API
+include::snippets/technology-preview.adoc[]
+
+The link:https://cluster-api.sigs.k8s.io/[Cluster API] is an upstream project that is integrated into {product-title} as a Technology Preview for Amazon Web Services (AWS) and Google Cloud Platform (GCP) clusters. You can use the Cluster API to create and manage machine sets and machines in your {product-title} cluster. This capability is in addition or an alternative to managing machines with the Machine API. 
+
+For {product-title} {product-version} clusters, you can use the Cluster API to perform node host provisioning management actions after the cluster installation finishes. This system enables an elastic, dynamic provisioning method on top of public or private cloud infrastructure.
+
+With the Cluster API Technology Preview, you can create compute machines and machine sets on {product-title} clusters for supported providers. You can also explore the features that are enabled by this implementation that might not be available with the Machine API.
+////
+// These use-cases may not really be relevant for the TP.
+
+[id="cluster-api-benefits_{context}"]
+== Benefits
+
+By using the Cluster API, {product-title} users and developers are able to realize the following advantages:
+
+* The option to use upstream community Cluster API infrastructure providers which might not be supported by the Machine API.
+
+* The opportunity to collaborate with third parties who maintain machine controllers for infrastructure providers.
+
+* The ability to use the same set of Kubernetes tools for infrastructure management in {product-title}.
+
+* Support for hub-and-spoke {product-title} clusters in which a management cluster can manage workload clusters that are running on different infrastructure providers.
```

Comment:
`hub-and-spoke` is not yet planned for openshift, it might be better to remove the line for now to avoid any questions from users